wormhole-connect/src/routes/sdkv2/route.ts

```diff
@@ -47,16 +47,10 @@ export class SDKv2Route {
 
   constructor(readonly rc: routes.RouteConstructor, routeType: Route) {
     this.TYPE = routeType;
-    // TODO: get this info from the SDK
-    if (routeType === Route.Relay) {
-      this.NATIVE_GAS_DROPOFF_SUPPORTED = true;
-      this.AUTOMATIC_DEPOSIT = true;
-    } else if (routeType === Route.NttRelay) {
-      this.AUTOMATIC_DEPOSIT = true;
-    } else if (routeType === Route.CCTPRelay) {
-      this.NATIVE_GAS_DROPOFF_SUPPORTED = true;
-      this.AUTOMATIC_DEPOSIT = true;
-    }
+
+    this.AUTOMATIC_DEPOSIT = rc.IS_AUTOMATIC;
+    this.NATIVE_GAS_DROPOFF_SUPPORTED = rc.NATIVE_GAS_DROPOFF_SUPPORTED;
```

Comment:
I realized that this will enable the gas slider for NTT and the NTT automatic route sets `NATIVE_GAS_DROPOFF_SUPPORTED = true`, but it's not *actually* supported by the relayer contracts at the moment. I will change this in the NTT SDK to be `false`.